packages/fluentui/projects-test/src/typings.ts

```diff
@@ -38,8 +41,8 @@ export async function typings() {
   logger(`✔️ Source and configs were copied`);
 
   await sh(`which yarn`);
-  await sh(`yarn --version`);
-  await sh(`yarn tsc --version`);
-  await sh(`yarn tsc --noEmit`, tmpDirectory);
+  await yarn(`--version`, yarnCache);
+  await yarn(`tsc --version`, yarnCache, tmpDirectory);
```

Comment:
Intended addition of `tmpDirectory`?

Reply:
The addition is `yarnCache`, not `tmpDirectory`